ingestion_tools/scripts/common/config.py

```diff
@@ -248,8 +228,10 @@ def resolve_output_path(self, key: str, obj: BaseImporter) -> str:
             "tiltseries": "{dataset_name}/{run_name}/TiltSeries",
             "gain": "{dataset_name}/{run_name}/Frames/{run_name}_gain",
             "frame": "{dataset_name}/{run_name}/Frames",
-            "rawtilt": "{dataset_name}/{run_name}/TiltSeries",
-            "collection_metadata": "{dataset_name}/{run_name}/TiltSeries",
+            "rawtilt": "{dataset_name}/{run_name}/Frames",
+            "collection_metadata": "{dataset_name}/{run_name}/Frames",
```

Comment:
should this have been part of [feat: Update ingestion configs with collection_metadata entities](https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/257)?

Reply:
While, this could have been, the updates to the existing paths were covered as a part of this issue.